enhancements/network/no_overlay_mode.md

```diff
@@ -0,0 +1,965 @@
+---
+title: no_overlay_mode
+authors:
+  - Riccardo Ravaioli
+reviewers:
+  - Peng Liu
+approvers:
+  - Jaime Caamaño
+  - Surya Seetharaman
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - Joel Speed
+creation-date: 2025-07-21
+last-updated: 2025-12-17
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/CORENET-6133
+see-also:
+  - https://github.com/ovn-kubernetes/ovn-kubernetes/blob/master/docs/okeps/okep-5259-no-overlay.md
```

Comment:
nit: would it be better to point to a specific commit? In case the upstream enhacement gets updates?